eblocker-icapserver/src/main/java/org/eblocker/server/app/DeviceProperties.java

```diff
@@ -87,7 +87,8 @@ public DeviceProperties(@Named("deviceProperties") String devicePropertiesPath)
                 properties.load(resource);
             }
         } catch (EblockerException e) {
-            LOG.info("Cannot load device properties from {}, using default values", devicePropertiesPath, e);
+            LOG.info("Cannot load device properties from {}, using default values", devicePropertiesPath);
+            LOG.debug("Exception from: Cannot load device properties ", e);
```

Comment:
If I take the message in the log seriously, then this problem is not so bad and the application can handle it. For this reason, I have packed the stacktrace into a log entry with debug so that it is not displayed by default.